tests/PHPStan/Analyser/AnalyserIntegrationTest.php

```diff
@@ -948,7 +948,9 @@ public function testBug7581(): void
 	public function testBug7903(): void
 	{
 		$errors = $this->runAnalyse(__DIR__ . '/data/bug-7903.php');
-		$this->assertNoErrors($errors);
+		$this->assertCount(2, $errors);
+		$this->assertSame('Comparison operation ">" between (0|(non-empty-array<int|string, 0|(non-empty-array<literal-string&non-falsy-string, 0|float>)|float>)|float) and 0 results in an error.', $errors[0]->getMessage());
```

Comment:
before this commit, the [`$ffffffjjjjjjccccc` variable at line 214](https://github.com/phpstan/phpstan-src/blob/b1d12a45af16a7c5df61069f713f9537fc6ac8c1/tests/PHPStan/Analyser/data/bug-7903.php#L214) was just mixed. after this commit we know more precisely its a oversized array